### ISSUE TITLE
Due Dates for Seminar 6 and Assignment 1

### ISSUE THREAD

**Issue by lucymosquera** [STAT540-UBC/Discussion#301]
Hello! I just wanted to ask about the following due dates as they don't seem to jive.

Seminar 6 is listed as being due on Fri Feb 22nd (Reading Week), but is listed as being done in class on Wed Feb 27th.

Assignment 1 (Q1-3) is listed as being due on Thursday Feb 28th, but it seems to require the PCA skills we'll learn on Feb 27th in seminar 6. 

Please let me know if these due dates are accurate.

**Comment by saramostafavi** [comment]
Hi Lucy,
For seminar 6, this is correct. You typically finish the seminars in class, but we give you until Friday to hand it in.
For Q1-Q3, that's a good point. PCA will be covered in class on Wed (Feb 13), and you'll have plenty of time to look over lecture notes yourself before the assignment's due date. (You can of course use the material from seminar 6 to help with your assignment.)

**Comment by lucymosquera** [comment]
Yes, I've noticed and appreciated the extra time! But so, if we do it in class on Wednesday Feb 27th, shouldn't it be due on Friday March 1st? Not the Friday before? Or is there a seminar during Reading week?

**Comment by saramostafavi** [comment]
Hi Lucy,
Ooops, you are right that there is no seminars during reading week! (sorry was looking at the wrong month in my calendar) We are fixing the dates for seminar 6 due dates. We'll get back to you all soon regarding this issue.

**Comment by saramostafavi** [comment]
Hi Everyone,
After investigating the issue raised by Lucy, now the due dates for seminar 6 on the website are corrected. The issue was that there was a sub-page where due date of seminar 6 was not correct. The correct due date for seminar 6 is Friday March 1.
